velox/exec/SharedArbitrator.cpp

```diff
@@ -148,7 +148,9 @@ SharedArbitrator::findCandidateWithLargestCapacity(
 }
 
 SharedArbitrator::~SharedArbitrator() {
-  VELOX_CHECK_EQ(freeCapacity_, capacity_, "{}", toString());
+  if (checkUsageLeak_) {
+    VELOX_CHECK_EQ(freeCapacity_, capacity_, "{}", toString());
```

Comment:
Can you add an error log if checkUsageLeak_ is false? The same memory manager case? Thanks!